torch/linalg/__init__.py

```diff
@@ -403,6 +403,97 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+svd = _add_docstr(_linalg.linalg_svd, r"""
+linalg.svd(input, full_matrices=True, compute_uv=True, *, out=None) -> (Tensor, Tensor, Tensor)
+
+Compute the singular value decomposition of either a matrix or batch of
```

Comment:
Analogous updates to the above needed here. For example:  "Compute" -> "Computes".